sphinx_wagtail_theme/layout.html

```diff
@@ -126,16 +126,18 @@
                         {% include "breadcrumbs.html" %}
                     </div>
                     <div class="col-sm-12 col-lg-auto mt-3 mt-lg-3">
-                        <noscript>
-                            <p>{{ _('JavaScript is required to toggle light/dark mode.') }}.</p>
-                        </noscript>
-                        <button id="wagtail-theme" class="btn btn-sm btn-light text-decoration-none" type="button">
-                            <span class="dark-only"><i class="fas fa-sun"></i> {{ _('Light mode') }}</span>
-                            <span class="light-only"><i class="fas fa-moon"></i> {{ _('Dark mode') }}</span>
-                        </button>
-                        {%- if pagename != "search" -%}
-                        {% include "source-buttons.html" %}
-                        {% endif %}
+                        <div class="d-flex flex-wrap" style="gap: 5px;">
```

Comment:
Thank you, but we don't want to use inline styles, as it's not good for Content Security Policy.

Reply:
Oh I see, I will change it then one thing, i noticed inline styling is being used in 3 filles:

https://github.com/wagtail/sphinx-wagtail-theme/blob/ebfbb71e90d1f5d6dabbaf431b7f50ef68247159/sphinx_wagtail_theme/extensions.html#L19

https://github.com/wagtail/sphinx-wagtail-theme/blob/ebfbb71e90d1f5d6dabbaf431b7f50ef68247159/sphinx_wagtail_theme/domainindex.html#L43

https://github.com/wagtail/sphinx-wagtail-theme/blob/ebfbb71e90d1f5d6dabbaf431b7f50ef68247159/docs/_templates/layout.html#L227

Should i clean them up as well ?

Reply:
Let's do that as a seperate fix, but feel free to raise a different PR. I've made a new issue for that. 

https://github.com/wagtail/sphinx-wagtail-theme/issues/305